src/status_im/accounts/login/core.cljs

```diff
@@ -40,8 +40,13 @@
 
 ;;;; Handlers
 (fx/defn login [cofx]
-  (let [{:keys [address password]} (accounts.db/credentials cofx)]
-    {:accounts.login/login [address password]}))
+  (if (get-in cofx [:db :hardwallet :whisper-private-key])
```

Comment:
`:whisper-private-key` sounds scary?

Reply:
@flexsurfer can you suggest better name? It's actually a private key for whisper

Reply:
it's not about name, just wonder if it's save to store private key in app-db

Reply:
@flexsurfer we will remove it from app-db after we pass it to status-go.